lib/elixir/pages/mix-and-otp/introduction-to-mix.md

```diff
@@ -176,6 +176,8 @@ iex> recompile()
 
 If anything had to be compiled, you see some informative text, and get the `:ok` atom back, otherwise the function is silent, and returns `:noop`.
 
+To go further, check out [`mix compile.elixir`](https://hexdocs.pm/mix/Mix.Tasks.Compile.Elixir.html) documentation.
+
```

Comment:
This is often in the first week of an Elixir dev journey, we don't need to mention `compile.elixir`.

```suggestion
```